### ISSUE TITLE
Wallet won't sync, coins are trapped!!!

### ISSUE THREAD

**Issue by Ashman113** [BitProfile/Xeth#47]
Is there any other wallet that is directly compatible with the Xeth skey file? So I can load up the file and access my coins? If not could you please make one of the older, pre-compiled stable versions available for download - the only available version on the website is 0.4.2 Beta.

I've stupidly sent coins to an address before Xeth had finished syncing. The 0.4.2 Beta wallet has now started refusing to sync so I am unable to access my coins to transfer them elsewhere. Currently Xeth will sync for around 5 minutes before peers drop to zero and a message flashes at the top of the screen saying it is unable to reconnect. Closing Xeth then killing the process with task manager (because just closing it doesn't actually kill it) and then restarting starts it syncing again but after about 5 minutes it does exactly the same thing again. After me continually doing this for 2 hrs it has now decided it doesn't even want to get past the loading screen anymore.  I've uninstalled it and reinstalled it and it's just doing the same thing again. 

I just need to get my coins out at this point.

**Comment by devktor** [comment]
Looks like your blockchain got corrupted. Deleting blockchain should fix it. Please tell me what OS do you have and I'll give you the path.

Have you used a stealth address? If you know the transaction hash, I can make you a tool to uncover/use it before synchronization.

**Comment by Ashman113** [comment]
Thanks, I'm using Windows 7. No I did not use a stealth address. Also is it normal to only ever have 4 peers? That's the most I've ever seen.

**Comment by devktor** [comment]
Then you can export the key and use it with MIST or any other wallet.

If you want to use MIST, it should see your key by default without doing anything. For other wallets please copy your keys from C:\Users\<your user name>\AppData\Roaming\Ethereum\keystore

**Comment by Ashman113** [comment]
Thanks, I've tried C:\Users<your user name>\AppData\Roaming\Ethereum\keystore but it is empty.

The only key I can find anywhere is a .skey file in Roaming\Xeth\Keys

I backed up my entire C drive before I tried uninstalling/reinstalling Xeth so I'll check my backup and see if there's anything in the \AppData\Roaming\Ethereum\keystore folder incase it got deleted during the uninstall.

**Comment by devktor** [comment]
.skey is for stealth addresses. Normal keys are in %APPDATA%/Ethereum/keystore
Have you tried to start MIST wallet? If it sees your address then everything is fine, just wait to have it synchronized.

Are you sure you didn't used a stealth address? Can you show me the transaction?

**Comment by Ashman113** [comment]
I initially created a stealth address but never used it, I went back and generated a normal address which is where I sent the ether to, that must be where the .skey file has come from then. 

I've downloaded MIST. Nothing was detected automatically when it opened, but I went back through my backup and found a file in the backed up Appdata\Roaming\Ethereum which I have just imported. I'm having issues getting MIST to start syncing but it does look like the file has been successfully imported as it is displaying the original address to which the ether was deposited. I'll report back when I've solved the syncing issues with MIST and hopefully have everything sorted out.
